src/lib/util/util.ts

```diff
@@ -9,15 +9,14 @@ import nodeFetch, { RequestInit, Response } from 'node-fetch';
 import { CLIENT_SECRET } from '../../../config';
 import ApiRequest from '../structures/api/ApiRequest';
 import ApiResponse from '../structures/api/ApiResponse';
-import { APIEmojiData, APIUserData } from '../types/DiscordAPI';
+import { APIUserData } from '../types/DiscordAPI';
 import { Events } from '../types/Enums';
 import { GuildSettings } from '../types/settings/GuildSettings';
 import { UserSettings } from '../types/settings/UserSettings';
 import { UserTag } from './Cache/UserTags';
 import { BrandingColors, Time } from './constants';
 import { REGEX_UNICODE_BOXNM, REGEX_UNICODE_EMOJI } from './External/rUnicodeEmoji';
 import { LeaderboardUser } from './Leaderboard';
-import { LLRCDataEmoji } from './LongLivingReactionCollector';
```

Comment:
is `LLRCDataEmoji` still used elsewhere or can it be purged entirely from the `LongLivingReactionCollector` file?

Reply:
Yup:

https://github.com/kyranet/Skyra/blob/1c9accc2ee56e1a8467f5240cf8d82b2fabaf179/src/lib/util/Games/ConnectFour/PlayerHuman.ts#L51